packages/build/tests/core/flags/fixtures/package/node_modules/netlify-plugin-test/index.js

```diff
@@ -0,0 +1,10 @@
+const {
+  execPath,
+  env: { TEST_NODE_PATH },
+} = require('process')
+
+module.exports = {
+  onInit() {
+    console.log(execPath === TEST_NODE_PATH)
```

Comment:
Just a small suggestion, how about we change this to:
```suggestion
    console.log(`expect '${execPath}' to equal '${TEST_NODE_PATH}'. Got '${execPath === TEST_NODE_PATH}'`)
```
Or something similar so one could just look at the snapshot to understand what is being tested (without having to look at the test).

Reply:
Yes, good point!